health_check.go

```diff
@@ -0,0 +1,53 @@
+package parse
+
+import (
+	"encoding/json"
+	"net/url"
+)
+
+const HealthCheckEndPoint = "/health"
+
+type healthCheckT struct {
+}
+
+func ServerHealthCheck() (map[string]string, error) {
+
+	body, err := defaultClient.doRequest(&healthCheckT{})
+	if err != nil {
+		return map[string]string{"status": "fail"}, err
+	}
+	data := map[string]string{}
+	if err := json.Unmarshal(body, &data); err != nil {
```

Comment:
Doesn't look like `data` gets used anywhere and the return value from the server is never actually checked. So, if the server returns `{"status": "fail"}`, this returns `{"status": "ok"}`.

It's been a long time since I've needed this library for anything, but I might recommend the following:

- return `nil, err` if the server returns an error / json fails to parse
- otherwise, just return whatever the server returns